app/views/movies/_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= form_with(model: @movie, local:true, method: request_method) do |form| %>
-  <% if @movie.errors.any? %>
+  <% if !@movie.nil? && @movie.errors.any? %>
```

Comment:
Lets use `@movie.present?` instead of `!@movie.nil?`. We should avoid `!`

Reply:
okay